finch/processes/wps_geoseries_to_netcdf.py

```diff
@@ -0,0 +1,200 @@
+from pywps import ComplexInput, ComplexOutput, FORMATS, LiteralInput
+
+import logging
+from pathlib import Path
+from .wps_base import FinchProcess
+from . import wpsio
+from .utils import log_file_path, write_log, dataset_to_netcdf
+from xclim.core.formatting import update_history
+import xarray as xr
+import numpy as np
+import cf_xarray.geometry as cfgeo
+import geopandas as gpd
+from urllib.parse import urlparse
+
+LOGGER = logging.getLogger("PYWPS")
+
+
+class GeoseriesToNetcdfProcess(FinchProcess):
+    def __init__(self):
+        inputs = [
+            ComplexInput(
+                "resource",
+                "Geospatial series",
+                abstract="Series of geographical features.",
+                supported_formats=[FORMATS.GEOJSON],
+                min_occurs=1,
+                max_occurs=1,
+            ),
+            LiteralInput(
+                "index_dim",
+                "Index dimension",
+                abstract="Name of the column in the data to be converted into the index dimension.",
```

Comment:
This should usually be time, right ? Eventually we could try to autodetect that column based on which can be converted to a datetime object.

Reply:
In most cases yes. I guess that timeseries will only have a single column convertible into datetime. I also guess that in the case of the column convertible to datetime it is _always_ the index of the timeseries (the guess has to work in both directions).

Reply:
Is it possible that the geojson would have three separate columns (e.g. year, month, day) that the service should concatenate into one?    In this case the input could be a list of dims?  Not sure if possible but just thinking bigger picture than just what geomet give us

Reply:
Oh boy, I guess you're right! We could have:
If a string : do as it does now
If a list of 3 strings : assume it means year-month-day
if not given : use first column that converts to datetime.

Seems we are trying to cover very specific cases here, how robust would it be?

Reply:
I think I would say "let's cross that bridge if and when we get to it"  ... was just thinking out loud really